ofrak_core/Makefile

```
@@ -18,7 +18,11 @@ test: inspect
 	$(PYTHON) -m pytest -n auto test_ofrak --cov=ofrak --cov-report=term-missing
 	(sleep 2; echo 1; sleep 2; echo i agree) \
 		| python3 -m coverage run --append --source ofrak -m ofrak license --force
-	fun-coverage --cov-fail-under=100
+	@if [ "$(shell uname -m)" = "x86_64" ]; then \
+		fun-coverage --cov-fail-under=100; \
+	else \
+		fun-coverage; \
+	fi
```

```suggestion
	fun-coverage --cov-fail-under=100 || ! (shell uname -m | grep x86)
```

Same suggestion applies to all other instances of this change across the various package Makefiles below. 

I think doing it this way is a bit less brittle, and will have slightly more descriptive console output (since we'll still see coverage result failures, they just won't stop future tests from running).

---

I think I may have made an error here – I don't think the `shell` part at the beginning of the subshell should be necessary...

---

d'oh! should have caught that by testing it... fixed now